code/app/build.gradle.kts

```diff
@@ -42,12 +42,15 @@ dependencies {
     implementation(libs.activity)
     implementation(libs.constraintlayout)
     implementation(libs.espresso.intents)
+    implementation("com.squareup.picasso:picasso:2.71828")
     testImplementation(libs.junit)
     testImplementation(libs.ext.junit)
     testImplementation(libs.core)
     testImplementation(libs.ext.junit)
     androidTestImplementation(libs.ext.junit)
     androidTestImplementation(libs.espresso.core)
+    testImplementation("org.mockito:mockito-core:4.0.0")
```

Comment:
Do we need Mockito? this commit does not add any new tests

Reply:
I plan to implment new tests with it so have there in case. I can get rid of it if I do not use it

Reply:
please get rid of it if it's not used in this commit